### DISCUSSION TITLE
Can you set the log level?

### DISCUSSION THREAD

**Discussion by czhealy** [KyleRoss/node-lambda-log#51] [Q&A]
I would love to be able to configure the logger at certain levels given my environment, for example, INFO level and above logs are great for qa/dev environments, but I would like to only log WARNING and above in pilot/production environments. Is this possible? Cloudwatch == $$$.

**Comment by kaihendry** [comment]
I noticed my `log.debug` is not showing up...

**Reply by obiwabrakenobi** [reply]
Is it possible only to show i.E. warnings?

**Comment by KyleRoss** [comment]
This is planned for the next major version of LambdaLog. Given that this package currently supports custom log levels, it's made it hard to try to implement this feature so I'm deprecating it and removing it in version 4. Given time constraints right now, I hope to have the next major version out before the end of this year.

You can accomplish similar functionality with custom log levels currently, but it requires a decent amount of configuration.

**Reply by obiwabrakenobi** [reply]
Hi, I just found the time to submit some small PR enabling to set log levels. What do you think? I'm more into typescript so I hope everything is in js shape

https://github.com/KyleRoss/node-lambda-log/pull/62

**Reply by 0Richard** [reply]
Hi, Any ideas on this logging-level will be released? I'm just starting out and keen to get a choice of what I log out in Lambda env var.

**Reply by KyleRoss** [reply]
@0Richard Sorry for the delay on this. I'm hoping to get version 4.0 wrapped up soon, life has just been crazy over the past few months.

**Comment by kaihendry** [comment]
My workaround is

```
if (process.env.STAGE === 'dev') {
    log.options.debug = true;
}
```
